### PR TITLE
Update defaults.md with export to Word docx

### PR DESCRIPTION
**Pull request by do-not-perceive-me**
Information about Word Reference docx and defaults file. Small addition of noting that the default files are found in Assets manager.

### CODE REVIEW

**Review by nathanlesage — CHANGES_REQUESTED** [review]

File: docs/en/core/defaults-files.md [hunk 1]
```diff
@@ -79,3 +79,47 @@ variables:
...
 You can also make use of this templating system yourself. If you want to write a custom template, you can add statements such as `$if(myvariable)$$myvariable$$endif$` and insert the variable `myvariable` into any frontmatter or defaults file, and it will be replaced with whatever value you set it to.
+
+
+### Microsoft Word Template (.docx)
+
+For export as a Word document, you need a .docx document that serves as a template. This file should be stored in the same folder as your markdown files and added to the Word export Defaults file in Assets Manager. 
+
+#### Reference Document 
+
+Use an existing Word document or download the reference.docx from Pandoc and change the styles. Your reference doc is a Word file that you format to your preferences. You need to adjust the Word Styles (Paragraph, Heading, Body Text, Footnote Text, etc) for it to apply.
```
[the remark this answers is not in the document]
Comment (do-not-perceive-me) [reply docs/en/core/defaults-files.md:90]: for the repository, I don't see the download directory option, I only have the option to download the whole pandoc repository as a .zip

image

[the remark this answers is not in the document]
Comment (nathanlesage) [reply docs/en/core/defaults-files.md:90]: Ouh, alright, then it's probably the GitHub extension I have installed. Then … nevermind. Feel free to adapt the wording so that it's correct (zipping that directory and renaming it from `.zip` to `.docx` will still work)

[the remark this answers is not in the document]
Comment (do-not-perceive-me) [reply docs/en/core/defaults-files.md]: I tried to do that and it didn't work for me. I think it worked in the past though, so I assume it'll work for others. I included it in the updated file anyways.

[the remark this answers is not in the document]
Comment (do-not-perceive-me) [reply docs/en/core/defaults-files.md]: This is more of a request, but maybe you could include a copy of the reference documents in the Zettlr or Zettlr docs repos, and then add a link to it in documentation, so that it's easier for people to find and download. 

I don't really have much experience with command lines or github, so personally, it was a complicated and time consuming experience in trying to find the reference document to edit. I couldn't figure out where Pandoc was saving the doc or what I needed to change for it to show up or how to download it from github.

File: docs/en/core/defaults-files.md [hunk 2]
```diff
@@ -81,17 +81,60 @@ variables:
...
+### Reference Documents (.docx, .odt, .pptx) 
 
-For export as a Word document, you need a .docx document that serves as a template. This file should be stored in the same folder as your markdown files and added to the Word export Defaults file in Assets Manager. 
+For export as Microsoft Word document, OpenDocument, or PowerPoint presentation, Pandoc uses a reference document that serves as a style template. This section is written in relation to Word docx, but the same should apply for either ODT or PPTX. 
 
-#### Reference Document 
+#### Producing the Reference Document
 
-Use an existing Word document or download the reference.docx from Pandoc and change the styles. Your reference doc is a Word file that you format to your preferences. You need to adjust the Word Styles (Paragraph, Heading, Body Text, Footnote Text, etc) for it to apply. 
+For best results, it is recommended to download the reference document directly from Pandoc. It should already be installed with Zettlr, but if not, see [Pandoc installation](https://docs.zettlr.com/en/installing-pandoc/) instructions.
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:90]: If you use Pandoc to spit it out on the command line, it does not necessarily constitute a "download", so I would rephrase to:

> For best results, it is recommended to retrieve the reference document directly from Pandoc. It comes bundled with Zettlr, but if not, see the Pandoc installation instructions.

File: docs/en/core/defaults-files.md [hunk 3]
```diff
@@ -81,17 +81,60 @@ variables:
...
-For export as a Word document, you need a .docx document that serves as a template. This file should be stored in the same folder as your markdown files and added to the Word export Defaults file in Assets Manager. 
+For export as Microsoft Word document, OpenDocument, or PowerPoint presentation, Pandoc uses a reference document that serves as a style template. This section is written in relation to Word docx, but the same should apply for either ODT or PPTX. 
 
-#### Reference Document 
+#### Producing the Reference Document
 
-Use an existing Word document or download the reference.docx from Pandoc and change the styles. Your reference doc is a Word file that you format to your preferences. You need to adjust the Word Styles (Paragraph, Heading, Body Text, Footnote Text, etc) for it to apply. 
+For best results, it is recommended to download the reference document directly from Pandoc. It should already be installed with Zettlr, but if not, see [Pandoc installation](https://docs.zettlr.com/en/installing-pandoc/) instructions. 
+
+##### Option 1: Using Pandoc from Command line or Terminal
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:92]: Since option 2 is definitely more cumbersome, I would suggest to remove that option and only leave this option.

File: docs/en/core/defaults-files.md [hunk 4]
```diff
@@ -81,17 +81,60 @@ variables:
...
 
-#### Reference Document 
+#### Producing the Reference Document
 
-Use an existing Word document or download the reference.docx from Pandoc and change the styles. Your reference doc is a Word file that you format to your preferences. You need to adjust the Word Styles (Paragraph, Heading, Body Text, Footnote Text, etc) for it to apply. 
+For best results, it is recommended to download the reference document directly from Pandoc. It should already be installed with Zettlr, but if not, see [Pandoc installation](https://docs.zettlr.com/en/installing-pandoc/) instructions. 
+
+##### Option 1: Using Pandoc from Command line or Terminal 
+
+In the command line: `pandoc -o "custom-reference.docx" --print-default-data-file reference.docx`
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:94]: Rephrase:

> In order to retrieve a reference document, run any of the following commands to create the file `custom-reference.docx`, `custom-reference.pptx`, or `custom-reference.odt`:

And then list all three, but in a codeblock

File: docs/en/core/defaults-files.md [hunk 5]
```diff
@@ -81,17 +81,60 @@ variables:
...
+
+To adjust the name or location of where the document is saved, you can change `"custom-reference.docx"` to include the absolute path: `"/Users/[user]/Downloads/project-reference.docx"`
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:96]: Since the path only applies to macOS, I would suggest to say the following (or something along these lines): " … to include the absolute path, e.g., for macOS you could use …"

File: docs/en/core/defaults-files.md [hunk 6]
```diff
@@ -81,17 +81,60 @@ variables:
...
+
+##### Option 2: Download from Pandoc Github repository
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:98]: Remove this option

File: docs/en/core/defaults-files.md [hunk 7]
```diff
@@ -81,17 +81,60 @@ variables:
...
+5. Open with Word
+
+This method can lead to errors, and it is recommended to follow Option 1. 
+
+##### Option 3: Finding documents produced by others
+
+Formatted Pandoc reference documents can be found on Github or elsewhere. Use these at your own risk. 
+
+Examples: 
+- https://github.com/hokorobi/pandoc-docx-reference/blob/master/reference.docx
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:115]: Remove these links as we cannot guarantee for their safety or that they won't turn harmful in the future. Feel free to add helpful google search terms instead so that users can search for themselves.

File: docs/en/core/defaults-files.md [hunk 8]
```diff
@@ -81,17 +81,60 @@ variables:
...
+- https://www.autodidacts.io/convert-markdown-to-standard-manuscript-format-odts-docs-and-pdfs-with-pandoc/ 
+
+
+#### Adjusting the style and format
+
+The reference document is a Word file that needs to be formatted to your preferences using Styles. 
+<img width="500" alt="image" src="https://user-images.githubusercontent.com/109297886/200125385-1894484c-ea68-4615-aca3-065e40d271d9.png">
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:122]: Remove this image tag. Instead, replace it with a default Markdown image tag (``) and also, put the image in the appropriate `img` subdirectory. I do not wish links to images uploaded via the web interface.

File: docs/en/core/defaults-files.md [hunk 9]
```diff
@@ -81,17 +81,60 @@ variables:
...
+
+You must update the formatting in the Styles Pane (Paragraph, Heading 1, Body Text, Footnote Text, etc.) for it to apply. Changing the text or other contents of the document has no effect unless you "Update to Match Selection" in Styles. Pandoc ONLY looks at the Styles format.
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:124]: Instead of uppercase ONLY, use bold or italic emphasis.

Also, feel free to provide a link to the Microsoft documentation for those people who have never used the styles pane.

File: docs/en/core/defaults-files.md [hunk 10]
```diff
@@ -81,17 +81,60 @@ variables:
...
+
+
+#### Adjusting the style and format
+
+The reference document is a Word file that needs to be formatted to your preferences using Styles. 
+<img width="500" alt="image" src="https://user-images.githubusercontent.com/109297886/200125385-1894484c-ea68-4615-aca3-065e40d271d9.png">
+
+You must update the formatting in the Styles Pane (Paragraph, Heading 1, Body Text, Footnote Text, etc.) for it to apply. Changing the text or other contents of the document has no effect unless you "Update to Match Selection" in Styles. Pandoc ONLY looks at the Styles format. 
+
+- More information about which Styles are supported in Pandoc [documentation](https://pandoc.org/MANUAL.html#options-affecting-specific-writers) under `--reference-doc=FILE`.
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:126]:
```suggestion
> More information about which styles are supported in Pandoc can be found in its [documentation](https://pandoc.org/MANUAL.html#options-affecting-specific-writers) under `--reference-doc=FILE`. 
```

File: docs/en/core/defaults-files.md [hunk 11]
```diff
@@ -81,17 +81,60 @@ variables:
...
+
+If you have an existing document that you want to use as template, you can use Word Organizer to copy the styles over. 
+ 
+1. Format > Style...
+2. Organizer... at bottom left
+3. On the right side, under `Normal.dotm (global template)`, close file
+4. Open file, find your desired document
+5. Select the styles on the right, and <- Copy it over to the current reference.docx
+
+- In depth explanation of [Word Organizer](https://www.officearticles.com/word/using_the_organizer_in_microsoft_word.htm)
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:136]: Move this link to the first line of this section so that "Word Organizer" links there instead of this.

File: docs/en/core/defaults-files.md [hunk 12]
```diff
@@ -81,17 +81,60 @@ variables:
...
+
+- More information about which Styles are supported in Pandoc [documentation](https://pandoc.org/MANUAL.html#options-affecting-specific-writers) under `--reference-doc=FILE`. 
+
+If you have an existing document that you want to use as template, you can use Word Organizer to copy the styles over. 
+ 
+1. Format > Style...
+2. Organizer... at bottom left
+3. On the right side, under `Normal.dotm (global template)`, close file
+4. Open file, find your desired document
+5. Select the styles on the right, and <- Copy it over to the current reference.docx
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:134]: Is the `<-` arrow accidental?

File: docs/en/core/defaults-files.md [hunk 13]
```diff
@@ -113,13 +156,18 @@ self-contained: true
  etc. 
 ```
 
-More options for the writer can be found [here](https://pandoc.org/MANUAL.html#options-affecting-specific-writers-1), including options for numbered headings, the table of contents, etc. 
+Zettlr will search for the reference document in the same directory as the Markdown files that you're exporting. If you want different formatting for different projects and files, it can be convenient to have a custom-reference.docx within each of those folders.
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:159]: Really? Never tried this myself.

File: docs/en/core/defaults-files.md [hunk 14]
```diff
@@ -113,13 +156,18 @@ self-contained: true
...
 
-#### Summary for Export as Word (.docx)
+#### Summary for Export as Word, Open Documents, or Powerpoint presentations
 
-1. Make a reference.docx in Word by adjusting the styles
-2. Place that reference doc in the same folder with Markdown files for this export
-3. Change the Defaults file in Assets Manager to include that file
-4. Adjust any other parameters in the Defaults Manager. 
-5. Export as Word Document
+1. Download the custom-reference.docx from Pandoc
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:167]:
```suggestion
1. Retrieve the appropriate reference document from Pandoc
```

File: docs/en/core/defaults-files.md [hunk 15]
```diff
@@ -113,13 +156,18 @@ self-contained: true
...
+2. In Word, change the format of that document using Styles Pane or Organizer
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:168]:
```suggestion
2. In Word, change the format of that document using Styles Pane or Organizer; analogously with OpenDocument documents
```

File: docs/en/core/defaults-files.md [hunk 16]
```diff
@@ -113,13 +156,18 @@ self-contained: true
...
+3. Place that .docx in the same folder with Markdown files for this export or get the absolute path name
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:169]:
```suggestion
3. Place that reference file in the same folder as the Markdown files for this export or get the absolute path name 
```

File: docs/en/core/defaults-files.md [hunk 17]
```diff
@@ -113,13 +156,18 @@ self-contained: true
...
-2. Place that reference doc in the same folder with Markdown files for this export
-3. Change the Defaults file in Assets Manager to include that file
-4. Adjust any other parameters in the Defaults Manager. 
-5. Export as Word Document
+1. Download the custom-reference.docx from Pandoc
+2. In Word, change the format of that document using Styles Pane or Organizer
+3. Place that .docx in the same folder with Markdown files for this export or get the absolute path name 
+4. Adjust the Defaults file in Zettlr Assets Manager to include that file name
+5. Adjust any other parameters in the Defaults Manager
+6. Export as Word Document in Zettlr
```
Comment (nathanlesage) [inline docs/en/core/defaults-files.md:172]:
```suggestion
6. Export using the appropriate exporter profile in Zettlr 
```

**Review by nathanlesage — CHANGES_REQUESTED** [review]
Some things, mainly wording, but other than that it looks pretty good! Thank you already for the work.

File: docs/en/core/defaults-files.md (hunk 8 above)
Comment (do-not-perceive-me) [reply docs/en/core/defaults-files.md:122]: How do I upload the image? I don't have access to it to directly upload, and I'm not sure how to do it as a push request

File: docs/en/core/defaults-files.md (hunk 12 above)
Comment (do-not-perceive-me) [reply docs/en/core/defaults-files.md:134]: No, I was trying to describe that the arrow should be pointing to the left so that you're copying it over in the correct direction, but maybe I should just add another screenshot? But it might look different in windows than mac.  
image

(I'll take a better screenshot and upload it properly once I figure out how

File: docs/en/core/defaults-files.md (hunk 13 above)
Comment (do-not-perceive-me) [reply docs/en/core/defaults-files.md:159]: Yes it definitely works! I have a different custom-reference.docx for each project, and as long as they're in the correct folder, Zettlr uses the right one, which is really convenient for all the different types of reports I need.

**Comment by nathanlesage** [comment]
I'm closing as this PR seems abandoned.
